lessons/ro/chapter_3.yaml

```diff
@@ -0,0 +1,156 @@
+- title: Memorie
+  content_markdown: >
+    Aplicațiile scrise în Rust au 3 zone de memorie unde este stocată informație:
+
+    * **memoria pentru date** - pentru date care sunt de dimensiune fixă și sunt **statice** (ex:
+    mereu disponibile pe toată durata rulării aplicației). 
+      Spre exemplu, textul din programul tău (ex: "Hello World!"): Memoria ocupată (bytes) de acest text este citită dintr-un singur loc în cod
+      deci poate fi stocat în această zonă de memorie. Compilatoarele fac o groază de optimizări 
+      pentru acest tip de date și folosirea lor în general este considerată foarte rapidă, pentru că
+      locația lor este cunoscută și fixă.
+    * **memoria pentru stivă (stack)** - pentru date declarate ca variabile în interiorul unei
+    funcții. 
+      Locația în memorie a acestor date nu se schimbă pe durata apelului funcției; datorită acestui lucru compilatoarele 
+      pot optimiza codul astfel încât datele din stivă se accesează foarte rapid.
+    * **memoria pentru alocare dinamică (heap)** - pentru informație care este creată în timp ce aplicația
+    rulează. 
+      Datele în această zonă de memorie pot fi adăugate, mutate, șterse, redimensionate, etc. Din cauza 
+      naturii sale dinamice, este în general considerată mai lentă, dar această zonă permite 
+      utilizări mult mai creative ale memoriei. Când adăugăm date în această zonă de memorie, 
+      numim această operație **alocare (de memorie)**. Când eliberăm date din această zonă de memorie, numim această operație 
+      **dealocare (de memorie)**.
+- title: Crearea datelor în Memorie
+  content_markdown: >
+    Când **instanțiem** o **structură** în codul nostru, aplicația creează
+    câmpurile de date unul lângă altul în memorie.
+
+
+    Instanțiem o structură specificând toate valorile câmpurilor în interiorul 
+
+
+    `StructName { ... }`.
+
+
+    Câmpurile unei structuri sunt accesate folosind operatorul `.`.
+
+
+    Detalii despre memorie în exemplul nostru:
+
+    * Textul din interiorul ghilimelelor este informație read-only (ex: "Ferris"), deci acesta
+    este 
+      pus în zona *memoriei pentru date*.
+    * Apelul funcției `String::from` creează o structură de tip `String` care este pus
+    unul 
+      lângă altul cu toate câmpurile structurii SeaCreature în *stivă*. Un String reprezintă text care poate fi modificat 
+      și face acest lucru prin intermediul următoarelor aspecte:
+      1. Alocă memorie pe *heap* pentru text unde va putea fi modificat
+      2. Stochează o referință la acea locație de pe *heap* în structura `String` 
```

Comment:
```suggestion
      2. Buffer-ul de caractere din interiorul structurii 'String' e stocat pe heap
```

Reply:
The buffer itself is not stored inside the String struct, just the reference of the buffer is present (it's quite misleading to say "din interiorul structurii" here).

I think this idea is included in the first point of this list (line 85) and the English version states the same.